### DISCUSSION TITLE
トラックボールが作動しない

### DISCUSSION THREAD

**Discussion by lain320** [Yowkees/keyball#348] [Q&A]
keyball61をガイドに従って組み立てましたが、トラックボールのみ作動しません。
過去のQ&A( #276 )などを参考にハンダの確認、ProMicroの入れ替え、導通確認など行いました。
導通確認では5色の矢印マークの導通確認したところ、水色のみ導通が確認できませんでした。
この場合どのように対処すればよいでしょうか？
![20230918_042034104_iOS](https://github.com/Yowkees/keyball/assets/145315044/aa7235a1-9ae0-48f4-85bc-b70688ce9a59)
![IMG_9801](https://github.com/Yowkees/keyball/assets/145315044/d10a45e6-03d0-4ffe-aa15-80b7d56f9e84)
![IMG_9802](https://github.com/Yowkees/keyball/assets/145315044/cc4f2dbb-4a68-4f7f-ba76-bfdaaff1c6b5)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。またお写真もありがとうございます。

水色矢印どうしが導通していないということで、
下記赤丸で示すジャンパが大きさが大きく、もしかしたらテンプラしていそうですので、半田付けし直してみてください。
また、ProMicro裏の"NCS"の刻印がある端子の半田付けもちょっと穴があいているような、形がいびつなので半田を追加しても良いかと思います。
![KB61_trackballfix](https://github.com/Yowkees/keyball/assets/84209772/e3ddf8f6-c105-4b71-9459-6d25e8dd819b)

上記を見直しても導通しない場合には、エナメル線かゴム被覆銅線で下記の赤点どうしを接続してしまうのが早いと思います。
以上、お手数ですがご対応お願いいたします。
![KB61_trackballfix2](https://github.com/Yowkees/keyball/assets/84209772/01cb6c30-2c6f-48e3-9418-646891cbca17)

**Reply by lain320** [reply]
ご回答ありがとうございます。
アドバイスいただいた半田付けをやり直しましたが導通確認できませんでした。
ProMicro裏の赤点とL字コンスルーの赤点は接続しなくても導通しており、ProMicro裏の赤点とNCSの刻印がある端子の導通を確認したところ導通確認できなかったため、L字コンスルーの赤点とNCSの刻印がある端子を結線することでトラックボールが作動するようになりました。

**Comment by Yowkees** [comment]
ご対応ありがとうございました。
無事に動くようになったようで安心しました。
今後ともどうぞよろしくお願いいたします。
